core/src/main/java/apoc/create/Create.java

```diff
@@ -220,6 +221,31 @@ public Relationship vRelationshipFunction(@Name("from") Node from, @Name("relTyp
         return Stream.of(new VirtualPathResult(from, rel, to));
     }
 
+    @Procedure
+    @Description("apoc.create.vPath")
+    public Stream<PathResult> vPath(@Name("path") Path path) {
+        return Stream.of(createVPath(path));
+    }
+
+    @Procedure
+    @Description("apoc.create.vPaths")
+    public Stream<PathResult> vPaths(@Name("paths") List<Path> paths) {
+        return paths.stream().map(this::createVPath);
+    }
+
+    private PathResult createVPath(Path path) {
+        final Iterable<Relationship> relationships = path.relationships();
+        final Node first = path.startNode();
+        PathImpl.Builder builder = new PathImpl.Builder(new VirtualNode(first, Iterables.asList(first.getPropertyKeys())));
```

Comment:
I'd rather avoid using this class, even though it's currently public.
What about introducing a `VirtualPath` class that implements the stable `Path` interface, similar to what has been done with `VirtualNode` and `VirtualRelationship`?

Reply:
Ok, changed with `VirtualPath` class